components/desktop/mate/mate-screensaver/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Carsten Grzemba
+#
+
+MATE_CATEGORY= APP
+BUILD_STYLE= configure
+include ../../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/mate.mk
+
+COMPONENT_NAME=		mate-screensaver
+COMPONENT_VERSION=	1.26.1
+COMPONENT_MJR_VERSION=	1.26
```

Comment:
You only need to set COMPONENT_MJR_VERSION and COMPONENT_MNR_VERSION. COMPONENT_VERSION is set from them in mate.mk.